src/routes/LocationPage/index.js

```diff
@@ -65,13 +69,19 @@ const FetchingContentWrapper = compose(
 )(ContentWrapper)
 
 class LocationPage extends React.Component {
+  static propTypes = {
+    isPdfDownload: PropTypes.bool.isRequired
+  }
+
   render () {
-    return (
-      <RichLayout>
-        <FetchingContentWrapper/>
+    if (this.props.isPdfDownload) {
+      return <FetchingContentWrapper isPdfDownload={true}/>
+    } else {
+      return <RichLayout>
+        <FetchingContentWrapper isPdfDownload={false}/>
       </RichLayout>
-    )
+    }
   }
 }
 
-export default LocationPage
+export default connect((state) => ({isPdfDownload: state.router.query.pdf !== undefined}))(LocationPage)
```

Comment:
Simplify to: `export default connect((state) => ({isPdfDownload: !!state.router.query.pdf}))(LocationPage)` and use function instead of lambda or replace all functions with lambdas.

Reply:
Done.